src/Compilers/Test/Core/CommonTestBase.cs

```diff
@@ -37,6 +39,18 @@ public enum Verification
         PassesOrFailFast = 1 << 4,
     }
 
+    public readonly record struct Verification(VerificationStatus Status, string? ILVerifyMessage = null, string? PEVerifyMessage = null)
```

Comment:
Please update a test to show how `ILVerifyMessage` and `PEVerifyMessage` are used in practice (with non-default values) and to exercise the new comparison logic.